app/uk/gov/hmrc/voapropertylinking/controllers/test/ReverseRoutes.scala

```diff
@@ -0,0 +1,41 @@
+// @GENERATOR:play-routes-compiler
```

Comment:
same here... did you somehow break build.sbt? looks like play plugin is misconfigured and outputs generated router code into our code base